crates/hdk/src/lib.rs

```diff
@@ -334,6 +334,27 @@ getrandom::register_custom_getrandom!(wasm_getrandom);
 /// @todo in the future grant secrets may be moved to lair somehow.
 pub mod capability;
 
+/// Signing a single chain entry between multiple participants.
+///
+/// The basic goal is to enable a kind of atomicity across multiple source chains
+/// in an environment where countersigners trust each other in some ways but not
+/// entirely. Countersigning provides several trust models, including nominating
+/// a single party to gather signatures, M of N signers, majority signing buckets,
+/// etc.
+///
+/// The integrity layer enforces very little other than the structure of a
+/// countersigned entry, to define the session parameters and uniquness and final
+/// signature set. Implementations are expected to drive countersigning sessions
+/// through co-ordinator zomes based on understanding both the expected network
```

Comment:
I've only seen `coordinator` as spelling in the docs so far.
```suggestion
/// through coordinator zomes based on understanding both the expected network
```